src/title-pane/index.tsx

```diff
@@ -55,19 +57,12 @@ export const TitlePane = factory(function TitlePane({
 		initialOpen,
 		onClose,
 		onOpen,
+		open: openProp,
 		theme: themeProp
 	} = properties();
 	const { content, title } = children()[0];
 
-	let open = icache.get('open');
-	let performTransition = false;
-	const existingInitialOpen = icache.get('initialOpen');
-	if (existingInitialOpen !== initialOpen) {
```

Comment:
I believe you'd still need the `initialOpen` equality check in here would you not?